internal/app/friend/mongodb_repo.go

```diff
@@ -28,11 +28,45 @@ func (r *MongoRepository) FindByID(ctx context.Context, id string) (*types.Frien
 	defer s.Close()
 	var friend *types.Friend
 	if err := r.collection(s).Find(bson.M{"id": id}).One(&friend); err != nil {
-		return nil, errors.Wrap(err, "failed to find the given friend from database")
+		return nil, errors.Wrap(err, "Failed to find the given friend from database")
```

Comment:
Error can be wrapped into another error hence should not be started with CAP

Reply:
Fixed